setup.py

```diff
@@ -21,7 +21,8 @@
         sources=["acsylla/_cython/cyacsylla.cpp"],
         include_dirs=[CPP_CASSANDRA_INCLUDE_DIR],
         extra_objects=[CPP_CASSANDRA_STATIC_LIB_DIR],
-        extra_compile_args=["-std=c++11"],
+        extra_link_args=["-g"],
```

Comment:
I guess that we should remove this, `-g` should be used only for debugging purposes.

Reply:
ups, sorry about that! i was trying to debug it (unsuccesfully :P)

Reply:
done